packages/in_app_purchase/README.md

```diff
@@ -211,3 +251,8 @@ editing any of the serialized data structs, rebuild the serializers by running
 `flutter packages pub run build_runner build --delete-conflicting-outputs`.
 `flutter packages pub run build_runner watch --delete-conflicting-outputs` will
 watch the filesystem for changes.
+
+This plugin is in beta. Please use with caution and file any potential issues
```

Comment:
What about adding a `## Contributing to this plugin` heading for these last 2 paragraphs?

Reply:
done